Fixed some more coding standard issue

sundaramrajendran opened the pull request:
- Mixed double and single quotes issue, updated as double quotes
- Updated identifiers as camel case
- Unnecessary use of fully qualified name 'Assert.assertNotNull' due to existing import 'org.testng.Assert'
- The field name indicates a constant but its modifiers do not, added final keyword to the constant
- Fields should be declared at the top of the class, before any method declarations, constructors, initializers or inner classes

mach6 commented on client/src/main/resources/templates/js/main.js:
```diff
@@ -431,31 +431,31 @@ function displaySource(src, title) {
         if(n > 0) {
-          $(pageAdditionalId + 'testcase-pagination ul .page-number').last().remove();
-          $(pageAdditionalId + "testcase-pagination ul .page-pre").after($('<li>').attr('data-index',n).addClass('page-number').append($('<a>').attr('href','#').append(n) ));
+          $(pageAdditionalId + "testcase-pagination ul .page-number").last().remove();
```
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Line is too long.](https://www.codacy.com/app/SeLion/paypal-SeLion/file/1307322591/issues/source?bid=2928425&fileBranchId=2951451#l458)

mach6 commented on client/src/main/resources/templates/js/main.js:
```diff
@@ -394,9 +394,9 @@ function displaySource(src, title) {
         if(asc === "1") {
-          arrowDirection = 'dropup';
+          arrowDirection = "dropup";
         }
```
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Line is too long.](https://www.codacy.com/app/SeLion/paypal-SeLion/file/1307322591/issues/source?bid=2928425&fileBranchId=2951451#l401)

mach6 commented on client/src/main/resources/templates/js/main.js:
```diff
@@ -330,17 +330,17 @@ function displaySource(src, title) {
-        $(id1).first().append('<ul><li id="' + newId + '" data-value="' + key + '" data-type="' + type + '" ' + leaf + '>' + key + '</li></ul>');
+    var id1 = id.replace(/\./g, "").replace(/\s+/g, "");
+    var newId = key.replace(/\./g, "").replace(/\s+/g, "") + type + uniqueId;
+    if ($(id1 + " ul").length === 0) {
```
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Line is too long.](https://www.codacy.com/app/SeLion/paypal-SeLion/file/1307322591/issues/source?bid=2928425&fileBranchId=2951451#l341)

mach6 commented on client/src/main/resources/templates/js/main.js:
```diff
@@ -330,17 +330,17 @@ function displaySource(src, title) {
+    if ($(id1 + " ul").length === 0) {
+        $(id1).first().append("<ul><li id='" + newId + "' data-value='" + key + "' data-type='" + type + "' " + leaf + ">" + key + "</li></ul>");
     } else {
-        $(id1 + ' ul').first().append('<li id="' + newId + '" data-value="' + key + '" data-type="' + type + '" ' + leaf + '>' + key + '</li>');
```
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Line is too long.](https://www.codacy.com/app/SeLion/paypal-SeLion/file/1307322591/issues/source?bid=2928425&fileBranchId=2951451#l343)

mach6 commented on client/src/main/resources/templates/js/main.js:
```diff
@@ -463,27 +463,27 @@ function displaySource(src, title) {
         if(n <= getNoOfPage(pageAdditionalId)) {
-          $(pageAdditionalId + 'testcase-pagination ul .page-number').first().remove();
-          $(pageAdditionalId + "testcase-pagination ul .page-next").before($('<li>').attr('data-index',n).addClass('page-number').append($('<a>').attr('href','#').append(n) ));
+          $(pageAdditionalId + "testcase-pagination ul .page-number").first().remove();
```
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Line is too long.](https://www.codacy.com/app/SeLion/paypal-SeLion/file/1307322591/issues/source?bid=2928425&fileBranchId=2951451#l479)

mach6 commented on client/src/main/resources/templates/js/main.js:
```diff
@@ -524,15 +524,15 @@ function displaySource(src, title) {
-      $(pageAdditionalId + 'testcase-page-info').html("Showing all rows");
+      $(pageAdditionalId + "testcase-page-info").html("Showing all rows");
     } else {
-      $(pageAdditionalId + 'testcase-page-info').html("Showing " + rtIndex + " to " + endIndex + " of " + rtFilterSearch.length + " rows");
```
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Line is too long.](https://www.codacy.com/app/SeLion/paypal-SeLion/file/1307322591/issues/source?bid=2928425&fileBranchId=2951451#l529)

mach6 commented on client/src/main/resources/templates/js/main.js:
```diff
@@ -545,7 +545,7 @@ function displaySource(src, title) {
     var pageLoaded = 0;
     var i = startNumber;
     while ((i <= mNoOfPage) && (pageLoaded < 5)) {
-      $(paginationId + "testcase-pagination ul .page-next").before($('<li>').attr('data-index',i).addClass('page-number').append($('<a>').attr('href','#').append(i) ));
```
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Line is too long.](https://www.codacy.com/app/SeLion/paypal-SeLion/file/1307322591/issues/source?bid=2928425&fileBranchId=2951451#l548)

mach6 commented on client/src/main/resources/templates/js/htmlreporter.js:
```diff
@@ -16,36 +16,36 @@
 $(document).ready(function () {
 
   "use strict";
```
why not just change this to `'use strict';` instead and keep the use of `'` everywhere in tact?

sundaramrajendran replied:
@mach6 Since we are from Java background I felt using " feels better compare to use '. Another point main.js also we used ", so better we will follow the same standard in all our JS files.

mach6 replied:
@sundaramrajendran Hmm. Okay. Either approach (old or new) is fine with me as long as there is consistency. That said, if we switch to `"` in javascript, I'd like to retain the use of `"` for html attributes in html markup, generated by jquery, or otherwise. This change set uses `'` for html attributes in several places where before it was `"`. Meaning, we should find those lines, and change to `\"` escaping now -- which is one of the reasons I've always leaned towards `'` instead of `"` usage in javascript -- but again, either approach is fine with me.

mach6 commented on client/src/main/resources/templates/js/main.js:
```diff
@@ -330,17 +330,19 @@ function displaySource(src, title) {
-        $(id1).first().append('<ul><li id="' + newId + '" data-value="' + key + '" data-type="' + type + '" ' + leaf + '>' + key + '</li></ul>');
+    var id1 = id.replace(/\./g, "").replace(/\s+/g, "");
+    var newId = key.replace(/\./g, "").replace(/\s+/g, "") + type + uniqueId;
+    if ($(id1 + " ul").length === 0) {
```
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Line is too long.](https://www.codacy.com/app/SeLion/paypal-SeLion/file/1314305759/issues/source?bid=2928425&fileBranchId=2951451#l341)

sundaramrajendran commented:
Done

sundaramrajendran committed:
Fixed some more coding standard issues

 - Mixed double and single quotes issue, updated as double quotes
 - Updated identifiers as camel case
 - Unnecessary use of fully qualified name 'Assert.assertNotNull' due to existing import 'org.testng.Assert'
 - The field name indicates a constant but its modifiers do not, added final keyword to the constant
 - Fields should be declared at the top of the class, before any method declarations, constructors, initializers or inner classes

mach6 commented:
this is merged
